src/app/layout/header/header.component.ts

```diff
@@ -59,7 +59,7 @@ export class HeaderComponent implements OnInit, OnDestroy {
   ) {}
 
   async ngOnDestroy(): Promise<void> {
-    this._subscription$.next();
+    this._subscription$.next(null);
```

Comment:
Why this change?

Reply:
Yeah, to update some libs i had to update rxjs. There was a breaking change, that Subject needs to emit value.